pkg/release/condition_set.go

```diff
@@ -0,0 +1,62 @@
+package release
+
+import (
+	"context"
+
+	"github.com/giantswarm/backoff"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	apiextensionsclient "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
I don't like that we have to pull k8s.io deps to release package. But there is a helm client anyway, so I'm not very sad.